README.md

```diff
@@ -154,6 +154,14 @@ In Java you can do so as follow:
     );
 ```
 
+## Looking up your logs
+
+When you open your console in Datadog, navigate to the Logs section, and in the search bar, type 
+`source:mobile`. This will filter your logs to only show the ones coming from mobile applications 
+(Android and iOS).
+
+![Datadog Mobile Logs](https://raw.githubusercontent.com/DataDog/dd-sdk-android/master/screenshot.png)
```

Comment:
We could use [GH relative link](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes#relative-links-and-image-paths-in-readme-files) to have this image rendered also on this feature branch. But what we do here should work fine after merge to `master`.